achilles/app/upgrades/empty_upgrade_handler.go

```diff
@@ -0,0 +1,16 @@
+package upgrades
+
+import (
+    "context"
+
+	upgradetypes "cosmossdk.io/x/upgrade/types"
+	"github.com/cosmos/cosmos-sdk/types/module"
+)
+
+// EmptyUpgradeHandler - should be used to create upgrade handler for gov software upgrades that require no migration
+func EmptyUpgradeHandler() upgradetypes.UpgradeHandler {
+    return func(ctx context.Context, plan upgradetypes.Plan, vm module.VersionMap) (module.VersionMap, error) {
+        // your logic here
+        return vm, nil
```

Comment:
I'm not 100% sure but i think you need to call run migrations here. 

`return mm.RunMigrations(ctx, cfg, vm)`

https://github.com/bze-alphateam/bze/blob/main/app/upgrades/v720/upgrades.go#L26